help/README.md

```diff
@@ -0,0 +1,48 @@
+# Welcome to New Help!
+Here are some instructions on how to get started with New Help...
+
+## How to contribute
+Expensify is an open source app, with its public Github repo hosted at https://github.com/Expensify/App.  The newhelp.expensify.com website is a part of that same open source project.  You can contribute to this helpsite in one of two ways:
+
+### The hard way: local dev environment
+If you are a developer comfortable working on the command line, you can edit these files as follows:
+
+1. Fork https://github.com/Expensify/App repo
+  * `...tbd...`
+2. Install Homebrew: https://brew.sh/
+3. Install `rbenv` using brew:
+  * `brew install rbenv`
+4. Install ruby v3.3.4 using
+  * `rbenv install 3.3.4`
+5. Set the your default ruby version using 
+  * `rbenv global 3.3.4`
+6. Install Jekyll and bundler gem 
+  * `cd help`
+  * `gem install jekyll bundler`
+7. Create a branch for your changes
+8. Make your changes
+9. Locally build and test your changes:
```

Comment:
I think you're missing a step before 9 to run `bundle install` which installs the gems from the Gemfile via bundler.